hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber - Double(secondNumber)
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber * Double(secondNumber)
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = Double(firstnumber) / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+
+    func stringComparison(firstString: String, secondString: String) {
+        if firstString > secondString {
+            print("\(firstString) > \(secondString)")
+        }
+        else if firstString < secondString {
+            print("\(firstString) < \(secondString)")
+        }
+        else {
+            print("\(firstString) = \(secondString)")
+        }
+    }
+    
+    func sumFour(number: Int) {
+        let firstNumber = number / 1000
+        let secondNumber = (number / 100) % 10
+        let thirdNumber = (number / 10) % 10
+        let fourthNumber = number % 10
+        let sumFour = firstNumber + secondNumber + thirdNumber + fourthNumber
+        print("Сумма \(number) = \(sumFour)")
+        
+    }
+    
+    func errorRecursion() {
+        print("error")
+        errorRecursion()
+    }
+    
+    func numberPow(_ firstNumber: Int, _ secondNumber: Int = 2) {
```

Comment:
It's better to have a verb as the first word if you can. For example, `powNumber`